src/services/cloneProjectTemplate.ts

```diff
@@ -0,0 +1,57 @@
+import { exec, getTemplatesDirectory } from 'helpers';
+import { projectTemplateRepositoryUrl } from 'config';
+import path from 'path';
+import fs from 'fs';
+import { Spinner } from 'cli-spinner';
+import fetch from 'node-fetch';
+
+const getRawFileUrl = (repositoryUrl: string, fileName: string): string =>
+  repositoryUrl
+    .replace('github.com', 'raw.githubusercontent.com')
+    .replace(/.git$/, `/master/${fileName}`);
+
+const getRemotePackageVersion = async (): Promise<string> => {
+  const response = await fetch(
+    getRawFileUrl(projectTemplateRepositoryUrl, 'package.json'),
+  );
+
+  const { version } = await response.json();
+
+  return version;
+};
+
+const getLocalPackageVersion = (projectTemplateSrc: string): string => {
+  const { version: currentVersion } = JSON.parse(
+    fs.readFileSync(path.join(projectTemplateSrc, 'package.json')).toString(),
+  );
+
+  return currentVersion;
+};
+
+export const cloneProjectTemplate = async (): Promise<void> => {
+  const projectTemplateSrc = path.join(
+    getTemplatesDirectory(),
+    'packageTemplate',
+  );
+
+  const directoryExists = fs.existsSync(projectTemplateSrc);
+
+  if (directoryExists) {
+    const localVersion = getLocalPackageVersion(projectTemplateSrc);
+    const remoteVersion = await getRemotePackageVersion();
+
+    if (localVersion === remoteVersion) return;
+
+    await exec(`rm -rf ${projectTemplateSrc}`);
```

Comment:
ideally, it would be nice to inform user that there's a new version of CRA available and the update process is on the way